StatBot/Models/DataModel.cs

```diff
@@ -25,8 +25,8 @@ public static List<Statistic> Statistics
         {
             get
             {
-                var db = new PostgresStatsRepository();
-                return db.GetAll().ToList();
+                var db = new PostgresStatsRepository();                
+                return db.GetAll().ToList().OrderBy(x => x.Id).ToList();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor 'Statistics' into a method, properties should not copy collections. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2365)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Drop 'ToList' from the middle of the call chain. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2971)